assistant/src/main/java/com/ibm/watson/assistant/v1/model/RuntimeResponseGenericRuntimeResponseTypeChannelTransfer.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.assistant.v1.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** RuntimeResponseGenericRuntimeResponseTypeChannelTransfer. */
+public class RuntimeResponseGenericRuntimeResponseTypeChannelTransfer
+    extends RuntimeResponseGeneric {
+
+  /** Builder. */
+  public static class Builder {
+    private String responseType;
+    private String messageToUser;
+    private ChannelTransferInfo transferInfo;
+    private List<ResponseGenericChannel> channels;
+
+    public Builder(
+        RuntimeResponseGeneric runtimeResponseGenericRuntimeResponseTypeChannelTransfer) {
+      this.responseType = runtimeResponseGenericRuntimeResponseTypeChannelTransfer.responseType;
+      this.messageToUser = runtimeResponseGenericRuntimeResponseTypeChannelTransfer.messageToUser;
+      this.transferInfo = runtimeResponseGenericRuntimeResponseTypeChannelTransfer.transferInfo;
```

Comment:
transferInfo is using a different class compared to ` runtimeResponseGenericRuntimeResponseTypeChannelTransfer.transferInfo`

Reply:
https://github.com/watson-developer-cloud/java-sdk/pull/1165/files#diff-6563a5ea2b8c37aee391b2eefce08e8d9ca81be8974c65639bb5404bd1da560dL85

transfer Info from the RuntimeResponseGeneric class

Reply:
One is `ChannelTransferInfo`, the other is `DialogNodeOutputConnectToAgentTransferInfo`